hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -485,13 +486,13 @@ static void add_final_stage_code(struct PixelShader *ps, struct FCInputInfo fina
     QString *b = get_input_var(ps, final.b, false);
     QString *c = get_input_var(ps, final.c, false);
     QString *d = get_input_var(ps, final.d, false);
-    QString *g = get_input_var(ps, final.g, false);
+    QString *g = get_input_var(ps, final.g, true);
 
     qstring_append_fmt(ps->code, "fragColor.rgb = %s + mix(vec3(%s), vec3(%s), vec3(%s));\n",
                        qstring_get_str(d), qstring_get_str(c),
                        qstring_get_str(b), qstring_get_str(a));
     /* FIXME: Is .x correctly here? */
```

Comment:
If we drop the `vec3().x` construct, we should remove this comment too.

Reply:
Oops! Will fix this.

Reply:
Done